lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,6 +19,23 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(text, str):
+        return None
+    tokens = []
+    word = []
+    only_letters = []
+    for elem in text.lower():
+        if elem.isalpha():
```

Comment:
use continue if this condition is not True

Reply:
After "if not elem.isalpha()" there should be the next step, which is now after else. How can I use continue here?